backend/common/services/sandbox_service.py

```
@@ -0,0 +1,231 @@
+            if not s3_key:
+                continue
+            try:
+                content = await s3_service.download_file(s3_key)
+                if content is None:
+                    logger.warning(f"[DAYTONA] S3 download returned None for {s3_key}")
+                    continue
+                sandbox_path = f"/home/daytona/data/{filename}"
+                success = await self.upload_file(sandbox_id, sandbox_path, content)
+                if success:
+                    uploaded += 1
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)